travis-init.sh

```diff
@@ -34,4 +34,17 @@ if [[ "$TRAVIS_PHP_VERSION" != "hhvm" &&
         echo "extension=libev.so" >> "$(php -r 'echo php_ini_loaded_file();')"
     fi
 
+    # install 'libev' PHP extension (does not support php 7)
```

Comment:
Copy pasta? :spaghetti:

Reply:
Yeah I went for a 🍕 but ended up with a 🍝 instead,